DAQ_Simulation.py

```diff
@@ -10,21 +10,49 @@
 port = 5000
 
 
+possible_data = [
+    # {
+    #     'id': '421',
+    #     'data': '03FFBA00776F'
+    # },
+    # {
+    #     'id': '422',
+    #     'data': '68090000'
+    # },
+    # {
+    #     'id': '421',
+    #     'data': '01FFC000777F'
+    # },
+    # {
+    #     'id': '421',
+    #     'data': '00FFF70077FF'
+    # },
+    # {
+    #     'id': '601',
+    #     'data': '00010000'
+    # },
+    # {
+    #     'id': '6B0',
+    #     'data': '00010B90C710416C'
+    # },
+    {
+        'id': '501',
+        'data': '0F900EBF02A0029F'
+    },
+]
```

Comment:
Would it be wise to include which CAN ID each of these are? For example, 421 is IMU data.

Reply:
Although I guess it is documented [here](https://docs.google.com/spreadsheets/d/195Y2cf9C2mrA6QCLuLJv3NHsCqTLi-3Wd07q4T5rXuM/edit#gid=0). Maybe we just need to clean that up instead.

Reply:
Perhaps we could leave a comment at the top explaining where to find the CAN IDs?

Reply:
These are just for simulating the DAQ data coming from the CAN bus, should we still try to document this here? I feel like it would be better documented somewhere else when we actually work on parsing the data.

Reply:
The point of the comment would be to better explain what data is being passed. When writing test code (or in this case, simulation code) I think it's always best to explain what data is being provided in order to give some background. But that's just me